Doc/library/asyncio-llapi-index.rst

```diff
@@ -0,0 +1,511 @@
+
+.. list-table::
+    :widths: 50 50
+    :class: full-width-table
+
+    * - :meth:`transport.close() <BaseTransport.close>`
+      - Close the transport.
+
+    * - :meth:`transport.is_closing() <BaseTransport.is_closing>`
+      - Return ``True`` if the transport is closing or is closed.
+
+    * - :meth:`transport.get_extra_info() <BaseTransport.get_extra_info>`
+      - Request for information about the transport.
+
+    * - :meth:`transport.set_protocol() <BaseTransport.set_protocol>`
+      - Set a new protocol.
+
+    * - :meth:`transport.get_protocol() <BaseTransport.get_protocol>`
+      - Return the current protocol.
+
+
+Transports that can receive data (TCP and Unix connections,
+read-pipes, etc).  Returned from methods like
+:meth:`loop.create_connection`, :meth:`loop.create_unix_connection`,
+:meth:`loop.connect_read_pipe`, etc:
+
+.. rubric:: Read Transports
+.. list-table::
+    :widths: 50 50
+    :class: full-width-table
+
+    * - :meth:`transport.is_reading() <ReadTransport.is_reading>`
+      - Return ``True`` if the transport is receiving.
+
+    * - :meth:`transport.pause_reading() <ReadTransport.pause_reading>`
+      - Pause receiving.
+
+    * - :meth:`transport.resume_reading() <ReadTransport.resume_reading>`
+      - Resume receiving.
+
+
+Transports that can Send data (TCP and Unix connections,
+write-pipes, etc).  Returned from methods like
+:meth:`loop.create_connection`, :meth:`loop.create_unix_connection`,
+:meth:`loop.connect_write_pipe`, etc:
+
+.. rubric:: Write Transports
+.. list-table::
+    :widths: 50 50
+    :class: full-width-table
+
+    * - :meth:`transport.write() <WriteTransport.write>`
+      - Write data to the transport.
+
+    * - :meth:`transport.writelines() <WriteTransport.writelines>`
+      - Write buffers to the transport.
+
+    * - :meth:`transport.can_write_eof() <WriteTransport.can_write_eof>`
+      - Return :const:`True` if the transport supports sending EOF.
+
+    * - :meth:`transport.write_eof() <WriteTransport.write_eof>`
+      - Close and send EOF after flushing buffered data.
+
+    * - :meth:`transport.abort() <WriteTransport.abort>`
+      - Close the transport immediately.
+
+    * - :meth:`transport.get_write_buffer_size()
+        <WriteTransport.get_write_buffer_size>`
+      - Return high- and low-water limits for write flow control.
+
+    * - :meth:`transport.set_write_buffer_limits()
+        <WriteTransport.set_write_buffer_limits>`
+      - Set new high- and low-water limits for write flow control.
+
+
+Transports returned by :meth:`loop.create_datagram_endpoint`:
+
+.. rubric:: Datagram Transports
+.. list-table::
+    :widths: 50 50
+    :class: full-width-table
+
+    * - :meth:`transport.sendto() <DatagramTransport.sendto>`
+      - Send data to the remote peer.
+
+    * - :meth:`transport.abort() <DatagramTransport.abort>`
+      - Close the transport immediately.
+
+
+Low-level transports abstraction over subprocesses.
```

Comment:
I generally prefer singular form for the first part of a compound noun: "transport abstraction".